bin/newman.js

```diff
@@ -65,6 +65,10 @@ program
         'Specify the Client SSL passphrase (optional, needed for passphrase protected keys).')
     .option('--ssl-extra-ca-certs <path>',
         'Specify additionally trusted CA certificates')
+    .option('--cookie-jar <path>',
+        'Specify custom cookie jar file path')
```

Comment:
This must be under 120 char length, new-line not required.

Also, should we mention that this expects a ToughCookie serialized jar here?